lib/node_modules/@stdlib/number/float64/base/exponent/lib/native.js

```diff
@@ -0,0 +1,46 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var addon = require( './../lib/addon.node' );
+
+
+// MAIN //
+
+/**
+* Returns the unbiased exponent of a double-precision floating-point number.
+*
+* @private
+* @param {number} x - input value
+* @returns {integer32} unbiased exponent
+*
+* @example
+* var v = exponent( 4.0 ); // => 100
```

Comment:
Not sure what `=> 100` is supposed to indicate.

Reply:
I'd just copy over the examples from `./main.js`.

Reply:
Binary representation of 4.0

Reply:
This incorrect, as `4.0` is a binary double-precision floating-point number, not an integer. What's shown is not how `4.0` is stored.

Reply:
In any event, replacing this example with the examples from `main.js` is preferred. Example parity is useful as it conveys to the reader that behavior for the JS and C APIs are the same.

Reply:
Ohh okay, got it.